gapis/api/gles/compat.go

```diff
@@ -562,6 +562,13 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 				log.E(ctx, "Error decompressing texture: %v", err)
 			}
 
+		case *GlTexBufferEXT:
+			if !version.IsES { // Strip suffix on desktop.
```

Comment:
Maybe `if version.AtLeastGL(3, 1) {` to be explicit? (I still don't think we've finalized the target GL version, but it would be nicer/safer to include it)